plugins/catalog-react/src/components/EntityOwnerPicker/EntityOwnerPicker.tsx

```diff
@@ -199,6 +199,7 @@ export const EntityOwnerPicker = (props?: EntityOwnerPickerProps) => {
                 setText(e.currentTarget.value);
               }}
               variant="outlined"
+              onBlur={() => setText('')}
```

Comment:
Can you add a test for this seeing as there some already for this component? You would add it to the `EntityOwnerPicker.test.tsx` file

Reply:
Im not super familiar with testing, so maybe im missing something easy.  This onBlur is only altering the state of the text.  I know you can test state changes with enzyme, but it doesnt appear enzyme is being used in testing of this project so not sure if I should add it for this test.  Im unaware of anyway to test this functionality with the react testing library.  There does not appear to be any current tests around user text input in any of the EntityOwner components.

Reply:
That's fair, I have a hard time with tests myself. So looking around I think there might be some ideas used by this test: https://github.com/backstage/backstage/blob/f6b10802ff60c1ade8c48cf7dd186ea9eb72a34f/plugins/catalog-graph/src/components/CatalogGraphPage/SelectedKindsFilter.test.tsx#L106. The key is that it uses `userEvent` from `@testing-library/user-event` which I think is what we can use to get that `onBlur` fired so we can check the state is cleared